packages/utils/src/ts-eslint/Rule.ts

```diff
@@ -220,20 +220,42 @@ interface RuleContext<
    * Returns the current working directory passed to Linter.
    * It is a path to a directory that should be considered as the current working directory.
    * @since 6.6.0
+   * @deprecated
```

Comment:
Come to think of it, it's probably best to explain why deprecated. What to you think about this?

```suggestion
   * @deprecated Use {@link `cwd`} instead.
```

Reply:
Yeah, looks great!